tests/storage/repository/filesystem/config/test_config_files.py

```diff
@@ -87,8 +88,12 @@ def test_parse_bindings(tmp_path: Path) -> None:
         path=study_path,
         version=-1,
         bindings=[
-            BindingConstraintDTO(id="bindA", areas=[], clusters=[]),
-            BindingConstraintDTO(id="bindB", areas=[], clusters=[]),
+            BindingConstraintDTO(
```

Comment:
Bad types: `areas` and `clusters` are sets, not lists. Use:

```python
BindingConstraintDTO(
    id="bindA",
    areas=set(),
    clusters=set(),
    time_step=TimeStep.HOURLY,
),
```

Reply:
Change the second section in the INI file to add a `type` option, for instance:

```ini
[bindB]
id = bindB
type = weekly
```

So that, you can check that the `frequency` is correctly handled.